### ISSUE TITLE
Напрашивается while 

### ISSUE THREAD

**Issue by gormih** [1thenikita/EdgesOfOurRussia#1]
https://github.com/1thenikita/EdgesOfOurRussia/blob/f3621c27cb25ed11b51d2a8c2650e088d24fa9de/api.py#L100

**Comment by 1thenikita** [comment]
Да, но чуть позже, как разберу завал по делам, буду изменять бота и обновлять.
